website/www/site/content/en/contribute/_index.md

```diff
@@ -50,53 +50,106 @@ There are lots of opportunities to contribute. You can for example:
+  to create issues or add comments. Only contributors can be assigned issues. If
+  you want to be assigned issues, a PMC member can add you to the project contributor
+  group.  Email the [dev@beam.apache.org](/community/contact-us)
+  to ask to be added as a contributor in the Beam issue tracker, and include your ASF Jira username.
+
+- [Beam Wiki Space](https://cwiki.apache.org/confluence/display/BEAM/Apache+Beam):
+  Anyone has read access. If you wish to contribute changes, please create an account and request edit access on the
+  [dev@beam.apache.org](https://beam.apache.org/community/contact-us) mailing list (include your Wiki account user ID).
+
+- Pull requests can only be merged by a
+  [Beam committer](https://home.apache.org/phonebook.html?pmc=beam).
+
+- [Voting on a release](https://www.apache.org/foundation/voting.html): Everyone can vote. Only
+  [Beam PMC](https://home.apache.org/phonebook.html?pmc=beam) members should mark their votes as binding.
+
+### Communication
+
+All communication is expected to align with the [Code of Conduct](https://www.apache.org/foundation/policies/conduct).
+
+Discussion about contributing code to Beam happens on the [dev@beam.apache.org mailing list](https://beam.apache.org/community/contact-us/). Introduce yourself!
+
+Questions can be asked on the [#beam channel of the ASF Slack](https://beam.apache.org/community/contact-us/). Introduce yourself!
+
+<a class="arrow-list-header" data-toggle="collapse" href="#collapseContributing" role="button" aria-expanded="false" aria-controls="collapseContributing">
+
+{{< figure src="/images/arrow-icon_list.svg">}}
+
+## Before You Begin
+
+</a>
 
 ### Prerequisites
 
- - A GitHub account.
- - A Linux, macOS, or Microsoft Windows development environment
- - Java JDK 8 installed
- - [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
-   changes locally.
- - For SDK Development:
-      - [Go](https://golang.org) 1.12 or later installed for Go SDK development
-      - Python 3.x interpreters. You will need Python interpreters for all Python versions supported by Beam.
-        Interpreters should be installed and available in shell via `python3.x` commands. For more information, see:
-        Python installation tips in [Developer Wiki](https://cwiki.apache.org/confluence/display/BEAM/Python+Tips#PythonTips-InstallingPythoninterpreters).
- - For large contributions, a signed [Individual Contributor License
-   Agreement](https://www.apache.org/licenses/icla.pdf) (ICLA) to the Apache
-   Software Foundation (ASF).
-
-
-### Configuration options
+- A [GitHub](https://github.com/) account.
+- A Linux, macOS, or Microsoft Windows development environment.
+- Java JDK 8 installed.
+- [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
+  changes locally.
```

Comment:
"testing this website changes" -> "testing changes to this website"

Reply:
changed